.eslintrc.js

```diff
@@ -473,5 +473,36 @@ module.exports = {
         'import/prefer-default-export': 'off',
       },
     },
+    {
+      /**
+       * Examples are for demonstration purposes and should not be considered a part of the library.
+       * They don't contain ESLint setup, so we don't want them to contain ESLint directives
+       * We do, however, want to keep the rules in place to ensure the examples are following
+       * a reasonably
```

Comment:
Seems to have been copied incompletely
```suggestion
       * a reasonably similar code style as the library.
```

Reply:
I saw you enabled auto-merge so I went ahead and commit this